pkg/kotsadm/postgres.go

```diff
@@ -62,6 +70,13 @@ func ensurePostgresStatefulset(deployOptions types.DeployOptions, clientset *kub
 		if err != nil {
 			return errors.Wrap(err, "failed to create postgres statefulset")
 		}
+
+		if deployOptions.UseHostNetwork {
```

Comment:
Pretty much the same comment as https://github.com/replicatedhq/kots/pull/299/files?utf8=%E2%9C%93&diff=unified&w=1#r373874918